lisa/base_tools/__init__.py

```diff
@@ -3,5 +3,6 @@
 from .sed import Sed
 from .uname import Uname
 from .wget import Wget
+from .yum_config_manager import YumConfigManager
 
-__all__ = ["Uname", "Sed", "Wget", "Cat", "Rpm"]
```

Comment:
Add it in the `__init__` under tools like other base tools, so it's easy to be used as normal tools.